cli/cluster/config.go

```diff
@@ -0,0 +1,161 @@
+package cluster
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	fmtPathNotMap   = "path %v is not a map"
+	fmtPathNotExist = "path %v does not exist"
+)
+
+// Config is a container for deserialized configuration.
+type Config struct {
+	paths any
```

Comment:
> Pls, add a comment about this member.

Added:

```Go
	// paths is a container implementation for the deserialized configuration.
	//
	// At this moment it is fully-compatible with yaml.v2 Marshal()/Unmarshal()
	// functions. So it could be marshaled and unmarshaled directly to YAML.
	// We may change it in the future.
```